hal/halx86/legacy/bussupp.c

```diff
@@ -829,30 +833,30 @@ HalpDebugPciDumpBus(IN PBUS_HANDLER BusHandler,
             SubClassName += strlen("\n\t00  ");
```

Comment:
Note: In the code here and above I'm not fully satisfied with the single alone '\n' ; I would like it to be function of `NEWLINE` instead; any help about what needs to be slightly adjusted (I suppose, with +-1 offsets or `>` -> `>=` types of changes) would be welcome.

Reply:
Doesn't `strlen(NEWLINE "\t00  ");` just work here? See also lines 820-821.

Reply:
If I remember correctly, a single `\n` was used here just for compactness.

Reply:
I've replaced remaining NEWLINE's in the second commit, please retest. I think there should be no problem.